packages/fe-fpm-writer/templates/filter/Controller.js

```diff
@@ -0,0 +1,21 @@
+sap.ui.define(["sap/ui/model/Filter", "sap/ui/model/FilterOperator"], function(Filter, FilterOperator) {
+    "use strict";
+    return {
+        <%- (typeof eventHandlerFnName !== 'undefined' && eventHandlerFnName) || 'itemsFilter' %>: function(sValue) {
+            switch (sValue) {
+                case "0":
```

Comment:
I know this is only an example implementation that gets overwritten by the user later on, but it will only work for filter properties of numeric  type, I guess (it would not work for the property from https://sapui5untested.int.sap.eu2.hana.ondemand.com/test-resources/sap/fe/core/fpmExplorer/index.html#/customElements/customFilterContent).  I wonder if we could not provide other templates, for other property types.

Reply:
Yes, this is used as an example controller file. In Guided Development any input can be used as a filtered property and the type is not checked. It is not in the scope of the current task and can be added afterwards with templates appropriate for the property types.